doc/building.md

```diff
@@ -546,6 +546,20 @@ Hotspot.
 Use `--with-libffi=<path>` if `configure` does not properly locate your libffi
 files.
 
+### libsleef
+
+libsleef, the [SIMD Library for Evaluating Elementary Functions](
+https://sleef.org/) is required when building libvmath.so on Linux+AArch64
```

Comment:
The conventional way we have refered to os/cpu combinations in the build documentation is like this: `Linux/aarch64`.

I also think you need to expand a bit that this is optional, and if you do not provide libsleef, the build will succeed but without the vector performance enhancements provided by libvmath.

Reply:
Thanks for the review! This sounds good to me. I will add it.